_posts/2015-12-24-two-column.markdown

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: 2 column Blog with Video
+date: 12/24/15
+homepage: grassrootsblog.org/about/
+download: https://github.com/GRBlog/Jekyll-2-Column-Theme/archive/master.zip
+author: Robert Edwardes
+thumbnail: 2-column.jpg
```

Comment:
The thumbnail image filename should be exactly same as the posts filename, but without the date prefix.